lib/ui/painting/multi_frame_codec.h

```diff
@@ -26,24 +26,31 @@ class MultiFrameCodec : public Codec {
   Dart_Handle getNextFrame(Dart_Handle args) override;
 
  private:
-  const std::unique_ptr<SkCodec> codec_;
-  const int frameCount_;
-  const int repetitionCount_;
-  int nextFrameIndex_;
-
-  // The last decoded frame that's required to decode any subsequent frames.
-  std::unique_ptr<SkBitmap> lastRequiredFrame_;
-  // The index of the last decoded required frame.
-  int lastRequiredFrameIndex_ = -1;
-
-  sk_sp<SkImage> GetNextFrameImage(fml::WeakPtr<GrContext> resourceContext);
-
-  void GetNextFrameAndInvokeCallback(
-      std::unique_ptr<DartPersistentValue> callback,
-      fml::RefPtr<fml::TaskRunner> ui_task_runner,
-      fml::WeakPtr<GrContext> resourceContext,
-      fml::RefPtr<flutter::SkiaUnrefQueue> unref_queue,
-      size_t trace_id);
+  struct State {
```

Comment:
class docstring

Reply:
Done. It's probably not in proper doxygen format, but this is a private class anyway.